Source/WebCore/loader/DocumentLoader.cpp

```diff
@@ -739,7 +739,8 @@ void DocumentLoader::willSendRequest(ResourceRequest&& newRequest, const Resourc
         if (!parentFrame)
             return completionHandler(WTFMove(newRequest));
 
-        if (!MixedContentChecker::frameAndAncestorsCanDisplayInsecureContent(*parentFrame, MixedContentChecker::ContentType::Active, newRequest.url())) {
+        if (!MixedContentChecker::frameAndAncestorsCanDisplayInsecureContent(*parentFrame, MixedContentChecker::ContentType::Active, newRequest.url())
```

Comment:
Ditto here, maybe we should do the shouldBlockInsecureContent check inside frameAndAncestorsCanDisplayInsecureContent